hw/ip/aes/model/aes.c

```diff
@@ -37,6 +37,11 @@ int aes_encrypt_block(const unsigned char *plain_text, const unsigned char *key,
     round_key[i] = full_key[i];
   }
   rcon = 0;
+  
+  // print input block and key
+  aes_print_block((const unsigned char *)key, 32);
+  aes_print_block((const unsigned char *)plain_text, 16);
```

Comment:
perhaps remove these. If we need to always print them, use `uvm_info to print them in the DV seq.